### PR TITLE
Update documentation of TaylorNModel

### PR DESCRIPTION
**Pull request by schillic**
I was not sure how to describe `order_bounds`.

### CODE REVIEW

File: src/TaylorN/TaylorN.jl [hunk 1]
```diff
@@ -6,14 +6,15 @@ degree(f::TaylorN) = f.order
 setindex!(f::TaylorN, x, i) = f.coeffs[i+1] = x
 
 doc"""
-A `Taylor1Model` represents a polynomial approximation to a function $f(t)$ of a single variable.
+A `TaylorNModel` represents a polynomial approximation to a function $f(t)$ of `N` independent variables.
```
Comment (lucaferranti) [inline src/TaylorN/TaylorN.jl:9]: I know you just updated, so it's not your fault, but I dont think that is correct, I would say a taylor model is a rigorous approximation of a function composed by a polynomial approximation **and** an interval bound on the error

**Comment by lucaferranti** [comment]
WoW! quite  a lot of damage for a docstring update! :rofl: 

Just kiding it fails on main too. @lbenet do you remember off the top of your head if some feature depending on 1.9 was recently introduced? Otherwise I can try to have a look at this at some point (this is not related to this MR though)

**Comment by lucaferranti** [comment]
relevant stacktrace

```
ERROR: LoadError: UndefVarError: RetAlloc not defined
Stacktrace:
 [1] getproperty(x::Module, f::Symbol)
   @ Base ./Base.jl:31
 [2] top-level scope
   @ ~/work/TaylorModels.jl/TaylorModels.jl/src/validatedODEs.jl:420
 [3] include(mod::Module, _path::String)
 ```

**Comment by lbenet** [comment]
The problems were introduced by merging https://github.com/JuliaDiff/TaylorSeries.jl/pull/312. TS was processing inconsistently the series of some functions for intervals, in the sense of not caring for the natural domain of the function. I started working on that, but got distracted. Sorry...

**Comment by schillic** [comment]
Given that the build failures are unrelated, is there anything you expect me to do here?

**Comment by lbenet** [comment]
Regarding `Taylor1Model` and `TaylorNModel`, these were some structs defined during early essays, which have been left there; they do not allow the functionality that has been implemented for `TaylorModel1` and `TaylorModelN`. So, I don't think it is worth using them... Sorry for the very late response, I just noticed this.

Regarding the failure of the tests, as I mentioned it is related to https://github.com/JuliaDiff/TaylorSeries.jl/pull/312; it seems to me that the problem is the tests we implemented for `TaylorModel1{TaylorN{T}}`. Again, I haven't had time to get into that, but I'll try to remedy soon.

**Comment by schillic** [comment]
Okay, then I close this PR.

**Comment by lbenet** [comment]
Thanks... and sorry for the veeery late response...

**Comment by schillic** [comment]
No problem. Regarding the other problem, you have a branch that seems to work (#146). Maybe you can just merge that? :)

**Pull request closed without merging**